docs/docs/examples/data_connectors/GoogleDriveDemo.ipynb

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
<p>NIT: this is better to be moved in the cell above -- our upgrade scripts just didn't handle this when we migrated from v0.9.42 to v0.10.1</p><p> </p><p>that is, <code></code><code>%pip install llama-index llama-index-readers-google</code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/12304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>